### PR TITLE
fix app: close vault creation panel if there is no more deposits

### LINKED ISSUE #202: After acknowledging 2 vaults, i get back to a "Create new vault" screens
Which tells me to make new deposits. We should just be back to home i think.

### PR DESCRIPTION
**Pull request by edouardparis**
After creating a vault the user is redirected
back to home panel if their is no more deposits

close #202

### CODE REVIEW

**Review by danielabrozzoni — COMMENTED** [review]
A nit: when you don't have more deposits and you click on "Close" you see the "Create vaults" window for a couple of ms, then it closes. It's fine, but it's a bit glitchy... Do you think that there's an easy way to make it not glitchy? Or instead, do you think it will be quickly fixed when we work on #46?

File: src/app/state/stakeholder.rs [hunk 1]
```diff
@@ -188,6 +189,14 @@ impl StakeholderCreateVaultsState {
     pub fn on_vault_select(&mut self, ctx: &Context, outpoint: String) -> Command<Message> {
         if let Some(selected) = &self.selected_vault {
             if selected.vault.outpoint() == outpoint {
+                if self.deposits.len() == 1
+                    && self.deposits[0].vault.outpoint() == outpoint
```
[the remark this answers is not in the document]
Comment (edouardparis) [reply src/app/state/stakeholder.rs:193]: Yes, we should expect that the selected vault is the last remaining in the deposits, but I still wanted to check.
Do you think I should remove it ?

[the remark this answers is not in the document]
Comment (danielabrozzoni) [reply src/app/state/stakeholder.rs:193]: No it's fine, you can leave it :)

**Comment by edouardparis** [comment]
>A nit: when you don't have more deposits and you click on "Close" you see the "Create vaults" window for a couple of ms, then it closes. It's fine, but it's a bit glitchy... Do you think that there's an easy way to make it not glitchy? Or instead, do you think it will be quickly fixed when we work on #46?

Yes, or we remove the "Secure vault" part from the vault state and implement a new state for the "create vault panel".

**Review by danielabrozzoni — APPROVED** [review]
ACK 08e865234826814b6d3248728ec7b7ae2da49165

**Pull request merged**